src/index.js

```diff
@@ -146,4 +146,48 @@ DateTime.prototype.minusBusiness = function({ days = ONE_DAY } = {}) {
   return this.plusBusiness({ days: -days });
 };
 
+/**
+ * @typedef BusinessDiffConfig
+ * @property {boolean} [includeEndDate=false] - include the end date in the calculation
+ * @property {boolean} [relative=false] - signs the return value as negative if end date is in the past
```

Comment:
If I think about how I use a calendar, I say things like 2 days ago (-2) so it doesn't make total sense to me that a difference would be absolute by default. Is there a reason for this? I remember being confused with the moment version of this plugin when I first saw that too.

Both [diff](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-diff) and [diffNow](https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-method-diffNow) are relative and I'd like to to keep this plugin similar to Luxon core concepts

Reply:
I think the rationale is that you already have methods for determining whether a date is in the past and it is _probably_ a less common scenario to need a negative value for a business day calculation.

That being said, consistency is a really strong argument so if you think it would be better defaulting relative to true I will support that.

Reply:
Ok cool, let's go with relative as default